### PR TITLE
docs(readme): updates readme

### PR DESCRIPTION
**Pull request by 2xAA**
Updates the README with the project's logo, shield.io badges and a few more sections.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,17 @@
 {
   "name": "modv",
   "productName": "modV",
-  "author": "vcync",
+  "author": "Sam Wray",
```
Comment (TimPietrusky) [inline package.json]: Why not leave it as `vcync`?

File: README.md [hunk 2]
```diff
@@ -1,58 +1,117 @@
...
 
-### Other Linux flavours
 
-Untested. NDI is provided by grandiose, our fork is here: [https://github.com/vcync/grandiose/](https://github.com/vcync/grandiose/)
-This fork of grandiose has other libndi supported platforms, however even on Ubuntu we needed the above libndi package to be installed.
+## License
 
-Let us know how you get on (good or bad) and we'll update the repo and docs accordingly.
\ No newline at end of file
+Copyright © 2020 [Sam Wray](https://github.com/vcync).
```
Comment (TimPietrusky) [inline README.md]: Why not as `vcync`?

**Review by TimPietrusky — CHANGES_REQUESTED** [review]
I have two tiny questions.

File: README.md (hunk 2 above)
Comment (2xAA) [reply README.md]: I used a generator for the basic template. Didn't realise it had changed this - I'll update!

File: package.json (hunk 1 above)
Comment (2xAA) [reply package.json]: I used a generator for the basic template. Didn't realise it had changed this - I'll update!

**Review by TimPietrusky — APPROVED** [review]
LGTM

**Pull request merged**